### DISCUSSION TITLE
2024-03-03-simple

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#181] [General]
# 我的极简生活经验 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Life想和大伙随便聊聊的极简主义，我挺喜欢这种生活方式，但并不是说每个人需要极简，有的人喜欢收藏，有喜欢购物买很多东西，让自己舒心的生活方式我觉得都挺好的。但假如你也喜欢简单生活方式，或者有点儿强迫症，那这些小建议可能适合你。

https://tw93.fun/2024-03-03/simple.html

**Comment by wuuuuuzy99** [comment]
关于这一条“减少网络中的被动消息”真的很赞同作者的看法，顺便想问一下，有没有什么管理自己账号的好方法，例如手机号，各种社交媒体账号，工作生活微信账号等

**Reply by tw93** [reply]
这里其实不需要太管理 尽量是一个id就好，然后基本上也不会下线，忘记密码有手机可以找回

**Comment by josangmi** [comment]
自己一个人好办，一家三口一点办法没有。根本无法改变，尤其是疫情过后，囤东西更多了。

**Comment by zzfly256** [comment]
搬了几次家，从 极简生活 到 被繁琐但价值不高的物件堆满 到 努力再次回归极简生活，深有所感

**Comment by breakTBB** [comment]
一直没一个比较好的RSS流程，之前用feedly+rsshub

**Reply by tw93** [reply]
可以试试 NetNewsWire

**Comment by plutotree** [comment]
iOS的桌面整理真的极其不方便，我加了很多目录之后现在还有13屏，总计400多个，现在变得基本无法整理了。安卓一般有多选拖动，还是方便管理的。

**Reply by tw93** [reply]
我常用的只有一屏幕，哈哈哈，你怎么这么多

**Reply by plutotree** [reply]
花了一天整理删除之后，终于“只”留下了6屏单app+1屏文件夹，共计250多个app。

**Comment by imjiaoyuan** [comment]
同样喜欢极简主义！很少看到类似的文章，支持一下。

**Comment by Ryanst** [comment]
👍 和我的做法基本一样：舍弃所有不需要不重要的；永远不要仅仅为了便宜而购买；只开了微信消息通知；用订阅+搜索获取内容，而不是算法推荐；定期整理 review

**Comment by fengyun2** [comment]
请问一下文章中的折叠车是哪个牌子和型号呢？

**Reply by tw93** [reply]
大行 P8 很好骑

**Comment by gagagazan** [comment]
这种生活方式，简直是另一个我🤣  
关于整理：开启整理动作可能会有点困难，但是一旦开始就有种不想停下来的感觉。可以约上家人/朋友一起，从而克服这个静摩擦力。

**Comment by warmeaf** [comment]
很赞，感谢分享👍

**Comment by morgantina721-arch** [comment]
正在整理电脑里的东西，通过这个过程讲脑子也清理一下

**Comment by leeekyrie** [comment]
哈哈这个电视墙一眼发现是邻居

**Comment by arthaks** [comment]
这么干净的文字，有个错别字，麻烦铁子改一下：‘我很喜欢AI，但有不太喜欢AI下产生的大量的粗制滥造的内容’  。  里面的 ‘有’ --> ‘又’ 。
